site/en/blog/automatically-pause-on-any-exception/index.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: 'layouts/blog-post.njk'
+title: Automatically pause on any exception
+description: >
+   In the DevTools, you can automatically pause on any exception (especially useful for uncaught exceptions).
+authors:
+  - umarhansa
+date: 2015-05-16
+updated: 2015-05-19
+---
+
+<figure>
+{% Img src="image/T4FyVKpzu4WKF1kBNvXepbi08t52/mcIwVmL8QGFVDI0rvGCA.gif", alt="Automatically pause on any exception", width="614", height="390" %}
```

Comment:
This links points to the GIF for the "Autocomplete for bracket notation" post

So https://developer.chrome.com/blog/automatically-pause-on-any-exception/ shows the wrong GIF

Reply:
I would create a pull request but I don't know the link to the correct GIF

Reply:
The correct GIF can be found in the original post in umaar.com

https://umaar.com/dev-tips/32-pause-on-exception/

Reply:
I've created an issue https://github.com/GoogleChrome/developer.chrome.com/issues/2932